src/lib.rs

```diff
@@ -481,7 +485,19 @@ mod test {
         assert_eq!(decoded_result.unwrap(), hash1);
     }  
 
-    #[cfg(feature = "bench")] 
+    #[test]
+    fn base64_error_on_empty() {
+        let decoded_result = ImageHash::from_base64("");
+        assert!(decoded_result.is_err());
+        /* FIXME: Figure out how to make the below run
+        match decoded_result {
+            Err(FromBase64Error::InvalidBase64Length) => assert!(true),
```

Comment:
Actually if you were to do that, you'd need to have this at the top level:

```rust

use serialize::base64::{ToBase64, FromBase64, Config, STANDARD};
// Needs to be fully qualified
pub use serialize::base64::FromBase64Error;
```

And then the `test` submodule can just have:

```rust
use serialize::base64::*;
```